academic-services/knowledge-exploration-service/reference-makes-command-line-tool.md

```diff
@@ -171,37 +197,288 @@ The input entities file Url prefix.
+
+`--GrammarFilePath`
+
+The compiled grammar binary file path. Interpret requires a compiled grammar binary
+
+### Interpret optional parameters
+
+`--NormalizeQuery`
+
+Whether normalization rules should be applied to the query before making interpretations. Defaults to true.
+
+`--AllowCompletions`
+
+Whether to generate interpretations assuming the query is a partial query that's not yet fully formulated. When set to true, better interpretations can be generated for scenarios such as auto-suggest. Defaults to false.
+
+`--Offset`
+
+The number of top interpretations to be skipped/excluded in the result set. Defaults to 0.
+
+`--Count`
+
+The number of top interpretations to be included in the result set. Defaults to 5.
+
+`--InterpretationEntityAttributes`
+
+A list of entity attributes, seperated by ','. Use '*' for all entity attributes. Each interpretation in the result set can include the top matching entities used for generating the interpretation. InterpretationEntityAttributes specifies which attributes of the top matching entities should be included in the result set. Defaults to "*".
+
+`--InterpretationEntityCount`
+
+The number of top matching entities to be included for each interpretation. Each interpretation in the result set can include the top matching entities used for generating the interpretation. InterpretationEntityCount specifies how many top matching entnties should be included in the result set. Defaults to 10.
+
+`--Timeout`
+
+Maximum amount of time in milliseconds allowed for command to complete before aborting the command. The interpret command will return the top interptations found in the allowed timedout. Defaults to 2000.
+
+## Evaluate command
+
+Evaluates a KES query expression and returns the top matching entities in the index(es).
+
+```cmd
+kesm Evaluate --KesQueryExpression
+              --IndexFilePaths
+              [--Attributes]
+              [--Skip]
+              [--Take]
+              [--OrderBy]
+              [--OrderByDescending]
+              [--Timeout]
+```
+
+### Evaluate required parameters
+
+`--KesQueryExpression`
+
+The KES [query expression](.\concepts-query-expressions.md) to use for selecting entities from the index(es).
+
+`--IndexFilePaths`
+
+The file path expression for specifying which index file(s) to use. Use wild card to specify multiple indexes e.g. './index.*.kes'
+
+### Evaluate optional parameters
+
+`--Attributes`
+
+A list of entity attributes to be included in the result set, seperated by ','. Use '*' for all attributes. Defaults to 0.
+
+`--Offset`
+
+The number of top entities to be skipped/excluded in the result set. Defaults to 0.
+
+`--Count`
+
+The number of top interpretations to be included in the result set. Defaults to 5.
+
+`--OrderBy`
+
+Name of the entity attribute to use for sorting/ordering the entities in the result set. By default, entities are sorted by descending entity weight (static rank). Defaults to "weight"
```

Comment:
>Defaults to "weight" [](start = 164, length = 20)

Remove this, you actually already said what the default is in the previous sentence :-)